README.md

```diff
@@ -1,21 +1,119 @@
-A fully featured gantt chart component built entirely in Javascript, CSS and AJAX. It is lightweight and there is no need of external libraries or additional images.
+[![Build Status](https://travis-ci.org/mariohmol/jsgantt-improved.svg?branch=master)](https://travis-ci.org/mariohmol/jsgantt-improved)
```

Comment:
just created `https://travis-ci.com/jsGanttImproved/jsgantt-improved` for this

Reply:
changed

Reply:
ok... i just saw about the docs folder.. did that